.github/workflows/release-connector.yml

```diff
@@ -58,6 +58,6 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         upload_url: ${{ steps.create_release.outputs.upload_url }}
-        asset_path: ./Sample Report.pbit
+        asset_path: ./Power BI Templates/General Report/SafetyCulture Report Template.pbit
```

Comment:
Is the SC report template the same as the sample report?

Reply:
So this is a bug fix. A previous commit renamed/moved the template in another folder and forgot to update this file